go.mod

```diff
@@ -110,6 +110,5 @@ replace (
 	github.com/gorilla/websocket => github.com/gorilla/websocket v1.4.2
 	k8s.io/client-go => k8s.io/client-go v0.24.2
 	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.23.5
-
```

Comment:
This change is not needed, when nothing else is changed in this file.

Reply:
ok